test/factories/interview_questions.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :interview_question do
+    id 224
+    interview_id 155
+    question_id 7
```

Comment:
What are those magic numbers?
Please read how to work with relations with FactoryGirl, and fix this.